extension/scripts/features/hospitalfilter/ttHospitalFilter.css

```diff
@@ -0,0 +1,28 @@
+#hospitalFilter.tt-container {
+	margin-top: 10px;
+}
+
+#time-filter,
+#level-filter {
+	margin-left: 15px;
+}
+
+/*noinspection CssInvalidPropertyValue*/
+.tt-mobile #time-filter,
+.tt-mobile #level-filter {
+	max-width: -moz-available; /* WebKit-based browsers will ignore this. */
+	max-width: -webkit-fill-available; /* Mozilla-based browsers will ignore this. */
+	max-width: fill-available;
+	margin-left: 15px;
+}
+
+#hospitalFilter .time-filter-info-wrap,
+#hospitalFilter .level-filter-info-wrap {
+	margin-left: 15%;
```

Comment:
I'm honestly not a fan of setting a margin to a %, as it mostly looks shit on other sizes.